rmgpy/reduction/reductionTest.py

```diff
@@ -62,9 +64,18 @@ def setUpClass(cls):
         cls.targets = targets
         cls.error = error
 
+        # load kinetic database and forbidden structures
+        rmg.database = RMGDatabase()
+        path = os.path.join(settings['database.directory'])
+
+        # kinetics family loading
+        rmg.database.loadKinetics(os.path.join(path, 'kinetics'),
+                                       kineticsFamilies=['R_Addition_COm'],
+                                       reactionLibraries=[])
+
         reactionModel = rmg.reactionModel
         initialize(rmg.outputDirectory, reactionModel.core.reactions)
-    
+
```

Comment:
Could you either undo the change or remove this line completely? (single blank line between methods)